File: Components/BlogML/Xml/BlogMLComment.vb
Imports System.Xml.Serialization

Namespace BlogML.Xml
 <Serializable>
 Public NotInheritable Class BlogMLComment
  Inherits BlogMLNode
  Private m_userName As String
  Private m_userEmail As String
  Private m_userUrl As String
  Private m_content As New BlogMLContent

  <XmlAttribute("user-name")>
  Public Property UserName As String

  <XmlAttribute("user-url")>
  Public Property UserUrl As String

  <XmlAttribute("user-email")>
  Public Property UserEMail As String

  <XmlElement("content")>
  Public Property Content As BlogMLContent

 End Class
End Namespace

File: Server/Core/BlogML/Xml/BlogMLComment.cs
using System;
using System.Xml.Serialization;

namespace DotNetNuke.Modules.Blog.Core.BlogML.Xml
{
  [Serializable]
  public sealed class BlogMLComment : BlogMLNode
  {
    private string m_userName;
    private string m_userEmail;
    private string m_userUrl;
    private BlogMLContent m_content = new BlogMLContent();

    [XmlAttribute("user-name")]
    public string UserName { get; set; }

    [XmlAttribute("user-url")]
    public string UserUrl { get; set; }

    [XmlAttribute("user-email")]
    public string UserEMail { get; set; }

    [XmlElement("content")]
    public BlogMLContent Content { get; set; }

  }
}